packages/paste-core/components/button/src/DestructiveLinkButton.tsx

```diff
@@ -43,6 +43,7 @@ const DestructiveLinkButton = React.forwardRef<HTMLButtonElement, DirectButtonPr
       <Box
         ref={ref}
         width={fullWidth ? '100%' : 'auto'}
+        textAlign="left"
```

Comment:
Because the ButtonContents is `display: flex`, we don't actually need these. It makes the alignment 100% dependent on the justify rule in the index.tsx file